server_api.py

```diff
@@ -776,17 +776,15 @@ def edit_image_data(json_obj):
                 return_msg["error"] = "user right is too low"
                 return return_msg
             #check self image
-            sql = ("SELECT user_id, type_id FROM image_data WHERE img_id=\"" + img_id + "\"")
-            pure_result = db.query(sql)
+            with ImageDao() as imageDao:
+                result = imageDao.getImgCheck(imgId=str(img_id))
             try:
-                if pure_result[0][0] != user_id and user_level < user_level_high_bound:
-                    db.close()
+                if result[0][0] != user_id and user_level < user_level_high_bound:
                     return_msg["error"] = "can not modify other user image "
                     return return_msg
-                img_type_id = pure_result[0][1]
+                img_type_id = result[0][1]
             except:
-                db.close()
-                return_msg["error"] = "no such image id : " + img_id
+                return_msg["error"] = "no such image id : {img_id}".format(img_id=img_id)
                 return return_msg
         
         
```

Comment:
Another chance to format the code!
Please help us remove new line in line 789.
Make sure that only one new line between code blocks
thanks!